src/components/UI/FormItems/TextField.js

```diff
@@ -3,24 +3,36 @@ import PropTypes from 'prop-types';
 import MUITextField from '@material-ui/core/TextField';
 import FormControl from '@material-ui/core/FormControl';
 import FormHelperText from '@material-ui/core/FormHelperText';
+import InputAdornment from '@material-ui/core/InputAdornment';
 
 const propTypes = {
   className: PropTypes.string,
   error: PropTypes.oneOfType([
     PropTypes.bool,
     PropTypes.string,
     PropTypes.number
-  ])
+  ]),
+  endAdornment: PropTypes.node
 };
 
 const defaultProps = {
   className: '',
-  error: false
+  error: false,
+  endAdornment: null
```

Comment:
Change this to `InputProps: null`. See below comment. Also update the `propTypes` above